apps/root/src/hooks/useOpenConnectModal.ts

```diff
@@ -0,0 +1,106 @@
+import { useConnectModal } from '@rainbow-me/rainbowkit';
+import React from 'react';
+import useUser from './useUser';
+import { UserStatus } from 'common-types';
+import { defineMessage, useIntl } from 'react-intl';
+
+const useOpenConnectModal = () => {
+  const { openConnectModal } = useConnectModal();
+  const user = useUser();
+  const intl = useIntl();
+
+  return React.useCallback(() => {
+    if (!openConnectModal) return;
+
+    openConnectModal();
+
+    setTimeout(() => {
```

Comment:
I know this solves a mounting issue, but when the component is transitioning 'up', the modal content changes:

https://github.com/Mean-Finance/dca-fe/assets/144699868/8169c800-3aac-4bb1-b284-0c692c265b9d